QoL/documentation upgrades and some typo fixes

nosnowfall opened the pull request:
This pull request addresses #1 with some minor changes:

- fix function name typo and wrong capitalization on the kilo prefix
- amend the `engineering_form` output to gracefully handle +/-/0 exponents
- add alias functions for more readable inline number formatting
- update the readme to reflect current functionality of the module

nosnowfall committed:
Update EngineeringNotation.py

fixed 2 typos, added 2 alias functions, and amended E+val printing format

nosnowfall committed:
Update README.md

updated readme to reflect function-based structure rather than old class-based structure

nosnowfall committed:
Update EngineeringNotation.py

fixed bad behavior on values less than 1m and with 0s after the decimal point, as well as omitted whitespace after strings with neither an SI prefix nor a unit

RandomGgames commented:
Hey thanks for the pull request! Let me take a detailed look through it, do some testing, and can push it through to main. From a quick glance, all looks great.

I made this a while ago just for printing values from a multimeter at work, so it wasn't super polished. I appreciate the help and glad you found some use in this!

RandomGgames committed:
Reverted removing trailing 0s

RandomGgames commented:
woops didn't mean to do that

RandomGgames committed:
I stop touching things...

Nope they work, keeping it.

RandomGgames committed:
Revert "Reverted removing trailing 0s"

This reverts commit 93289336e221a7a45e49e0efe8058fc4bcd5b56f.

RandomGgames committed:
Merge branch 'main' of https://github.com/nosnowfall/EngineeringNotation into pr/2

RandomGgames committed:
Update EngineeringNotation.py

I apparently don't know how to press buttons on my computer. Keeping this. Sending it. Good enough.

nosnowfall commented:
thanks for taking a look!

ps I was there when you wrote it

RandomGgames commented:
:o but who?

RandomGgames commented:
Could it be? The man? The myth? The legend?
